crates/iota-sdk-types/src/lib.rs

```diff
@@ -196,7 +196,7 @@ macro_rules! def_is {
 }
 
 #[macro_export]
-macro_rules! def_is_as_opt {
+macro_rules! def_is_as_into_opt {
```

Comment:
Sorry but it should be `def_is_as_opt_into`, opt separated from as doesn't make sense

Reply:
Touché